models/repo.go

```diff
@@ -479,6 +488,41 @@ func (repo *Repository) mustOwner(e Engine) *User {
 	return repo.Owner
 }
 
+func (repo *Repository) getOwnerName(e Engine) error {
+	if len(repo.OwnerName) > 0 {
+		return nil
+	}
+
+	if repo.Owner != nil {
+		repo.OwnerName = repo.Owner.Name
+		return nil
+	}
+
+	u := new(User)
+	has, err := e.ID(repo.OwnerID).Cols("name").Get(u)
```

Comment:
```
has, err := e.Table("repository").Select("owner_name").Where("id=?", repo.OwnerID).Get(&repo.OwnerName)
```

Reply:
@lunny do you want me to submit PR to change this to master first?

Reply:
Do you mean something like:
```
has, err := e.Table("user").Select("name").Where("id=?", repo.OwnerID).Get(&repo.OwnerName)
```

Reply:
@lafriks the currently code is OK. I only want to say there is another way to do that.

Reply:
ok, thanks, I will keep in mind that